pyquil/api/_quantum_computer.py

```diff
@@ -693,7 +698,8 @@ def _get_9q_square_qvm(
     name: str,
     noisy: bool,
     qvm_type: str,
-    timeout: float,
+    compiler_timeout: float,
```

Comment:
Can we make as many of these keyword-only as possible (for all of these factories)? It's a private method but that will still keep things aligned, would be easy to otherwise mix up these timeouts

Reply:
Will do, great point